requirements/app/base.txt

```diff
@@ -13,3 +13,4 @@ beautifulsoup4<=4.8.2
 inquirer>=2.10.0
 psutil<5.9.4
 click<=8.1.3
+lightning_api_access@ git+https://github.com/Lightning-AI/LAI-API-Access-UI-Component@main
```

Comment:
Note that this won't be installed when you install lightning from PyPI

Reply:
Because it's a github url?

Reply:
Changed it to a requirement